src/Mod/Material/Gui/CMakeLists.txt

```diff
@@ -154,7 +154,11 @@ SET(MatGuiImages
     Resources/images/default_image.png
 )
 
-add_library(MatGui SHARED ${MatGui_SRCS} ${MatGuiIcon_SVG} ${MatGuiImages})
+SET(Material_Ui_Files
+    Resources/ui/materials-editor.ui
+)
+
+add_library(MatGui SHARED ${MatGui_SRCS} ${MatGuiIcon_SVG} ${MatGuiImages} ${Material_Ui_Files})
```

Comment:
@davesrocketshop  this is having some weird results for me when installing what I compile and I'm also seeing this happening in CI:
```
-- Installing: /usr/local/share/Mod/Material/Resources/icons/preview-rendered.svg
-- Installing: /usr/local/share/Mod/Material/Resources/icons/preview-vector.svg
-- Installing: /usr/local/share/Mod/Material/Resources/icons/table.svg
-- Installing: /usr/local/share/Mod/Material/Resources/images/default_image.png
-- Installing: /home/runner/work/FreeCAD/FreeCAD/build/release/Mod/Material/Resources/ui/materials-editor.ui
-- Installing: /usr/local/Mod/Material/Init.py
-- Installing: /usr/local/Mod/Material/InitGui.py
-- Installing: /usr/local/Mod/Material/importFCMat.py
-- Installing: /usr/local/Mod/Material/MaterialEditor.py
```
see how it's writing to an user directory when installing. My guess would be that `${Material_Ui_Files}` should not be in this line since it's not a library and it later gets installed anyways

Reply:
nvm, the issue is that it is later set to install that file to the build directory